react/features/base/toolbox/components/AbstractToolboxItem.js

```diff
@@ -7,17 +7,17 @@ export type Styles = {
     /**
      * Style for the item's icon.
      */
-    iconStyle: Object,
+    iconStyle: Object | Array<*>,
```

Comment:
Not sure what flow accepts, but shouldn't this be Array<Object>?

Reply:
We want to accept either a single style or an array of them, hence the "or".

Reply:
Sorry, github trimmed my comment, what I meant Array < Object >

Reply:
I couldn’t get flow to shut up :-(

Reply:
I use the same type markup in the indicators now, let me spend a bit getting back on this

Reply:
Ok, so the issue is the following: You allow the [icon|label]Style to be an Object or an Array < Object > but then in the AbstractButton you also do the following assignment:
iconStyle: [ buttonStyles.iconStyle, disabledStyles.iconStyle ]
where, if iconStyle is an Array < Object > , it ends up as an Array < Array < Object > >. And that's why it accepts Array < * > but not Array < Object >

So flow shouldn't shut up as it's warning you about a possible bug ;)

Reply:
Btw pls take a look at this function as I was dealing with something similar recently: https://github.com/zbettenbuk/jitsi-meet/blob/fe1447b5551e6a879f0a7c0c9ffbb6a24376e5a6/react/features/base/styles/functions.js#L76